app/main.py

```diff
@@ -2,5 +2,18 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+
+    cache_dict = {}
+
+    def wrapper(*args) -> int:
```

Comment:
Incorrect annotation, function returns not int and good to add an annotation, that this is a wrapped function. 
```suggestion
    @wraps(func)
    def wrapper(*args) -> Callable:
```

Reply:
fixed it